src/files/filesloader.ts

```diff
@@ -92,9 +92,16 @@ export class FilesLoader {
                     dot: true,
                     ignore: this.configStore.ignoredPathGlobs,
                     realpath: true,
+                    strict: false,
```

Comment:
Just for my understanding, the glob find will continue on error to try to find a possible match correct?
Before this (due to the bug you mentioned) it would stop abruptly if an error occurred even if everything had not yet been searched?

Reply:
Yep, that's right. With strict true it dies if it hits an error.

Reply:
Excellent, merging this should go out in the 2.9.0 release whenever I get around to finish a couple of the other items.